Frameworks/CoreFoundation/Base.subproj/CFFileUtilities.c

```diff
@@ -988,7 +988,8 @@ CF_PRIVATE CFIndex _CFStartOfLastPathComponent(UniChar *unichars, CFIndex length
         return 0;
     }
     for (idx = length - 1; idx; idx--) {
-        if (IS_SLASH(unichars[idx - 1])) {
+        // WINOBJCchange - handle cases of multiple trailing slashes
```

Comment:
looks like I undid this partway, oops

---
In reply to: [76706890](https://github.com/Microsoft/WinObjC/pull/934#discussion_r76706890) [](ancestors = 76706890)